authors/apps/notifications/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from ..authentication.models import User
+
+
+class Article(models.Model):
+    """This class creates the Article model"""
+    title = models.CharField(max_length=50, null=False, unique=True)
+    author = models.ManyToManyField(User)
+    description = models.CharField(max_length=250, null=False, default="")
+    body = models.TextField(null=False, default="")
+    read_time = models.PositiveIntegerField(default=1)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        """Returns a human readable representation of the model instance"""
+        return "{}".format(self.title)
+
```

Comment:
Work on PEP 8

Reply:
I am not getting what specific PEP8 violation I should fix here

Reply:
We should have 2 blank lines before a class.